.config/nginx/dev/nginx.conf

```diff
@@ -67,7 +67,7 @@ server {
         auth_basic off;
     }
 
-    location ~ ^/(_?_next.*|entities/?$) {
+    location ~ ^/($|_?_next.*|entities/?$) {
```

Comment:
@mattfredericksen, could you add a clarifying comment above this (indicating that it routes requests for the homepage and the entities page to the Next.js server)?